api/server.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+
+const Boardgames = require('../models/boardgamesModel');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/all', async (req, res) => {
```

Comment:
while it's not strictly necessary to stick with the `/{resources}` pattern, the convention really does help you mentally keep a lock on what you're dealing with